app/routes/items/new.js

```diff
@@ -162,6 +162,5 @@ export default AuthorizeRoute.extend(GradeMixin, {
     controller.set("defaultCondition", this.getDefaultCondition());
     controller.set("valueHkDollar", +defaultValue.value_hk_dollar);
     controller.set("defaultValueHkDollar", +defaultValue.value_hk_dollar);
-    this.setupPrinterId(controller);
```

Comment:
removed this as it is also called [here](https://github.com/crossroads/stock.goodcity/pull/854/files#diff-582238b7d4945ca1309bf7b22d97b16bR74)